src/lib/pages/my-module-verification-details/index.tsx

```diff
@@ -37,23 +38,49 @@ const MyModuleVerificationDetailsBody = ({ taskId }: { taskId: string }) => {
       />
     );
 
+  if (data.task.status === MoveVerifyTaskStatus.NotFound && !verifyModuleTask)
+    return (
+      <EmptyState
+        imageVariant="not-found"
+        heading="Verification Details is Unavailable"
+        withBorder
+        message=""
+      >
+        <Box>
+          <Text color="text.dark">
+            We’re unable to access the page you requested due to one of the
+            following possibilities:
```

Comment:
one line

Reply:
It's a linter. It will display in 1 line anyway.